Cargo.toml

```diff
@@ -1,3 +1,9 @@
+[patch.crates-io]
+#tokio = { git = "https://github.com/rjzak/tokio", branch = "master" }
+#tokio-macros = { git = "https://github.com/rjzak/tokio", branch = "master" }
+#tokio-util = { git = "https://github.com/rjzak/tokio", branch = "master" }
```

Comment:
There should either be a comment about why this is commented-out with a reference to a technical debt issue or these commented-out blocks should not be here